components/list/list.css

```diff
@@ -0,0 +1,51 @@
+.list {
+  min-width: 10%;
```

Comment:
is min-width dependent on parent container and will the component be always used with parent? otherwise the code might break if it is not clear 10% of what.

Reply:
width of x% is always x times parent width.